eth-benchmark-tests/src/jmh/java/tech/pegasys/teku/benchmarks/README.md

```diff
@@ -0,0 +1,5 @@
+# JMH Micro benchmarks
+
+These benchmarks can be  easily run in Intellij by installing the JMH plugin.
```

Comment:
Add at least way to run all without intellij: `./gradlew :infrastructure:kzg:jmh`. We have external contributors, people who are not used ot java

Reply:
ist a standard java concept i thought, i was borderline not going to put anything in, but i guess we can put that in (i've never run it like that, i'll have to check)